tests/testcore/functional_test_suite.go

```diff
@@ -25,12 +25,18 @@
 package testcore
 
 import (
-	"testing"
+	"go.temporal.io/server/common/testing/taskpoller"
+)
+
+type (
+	FunctionalTestSuite struct {
+		FunctionalTestBase
 
-	"github.com/stretchr/testify/suite"
+		TaskPoller *taskpoller.TaskPoller
+	}
 )
 
-func TestFunctionalSuite(t *testing.T) {
-	t.Parallel()
```

Comment:
Huh, did this have an effect before? If so, is it okay to remove it?

Reply:
I actually deleted entire file. Don't know why git thinks it is a change :-). There were no tests in this file. I think it is just some leftover after merging everything back to `_test.go` files.